### ISSUE TITLE
Weird SEGA logo while starting new save

### ISSUE THREAD

**Issue by SafeStrawberry89** [Rubberduckycooly/Sonic-1-2-2013-Decompilation#119]
While trying to start a new game, the SEGA logo appears on the top left corner all bugged out. The menu and the rest works fine except this. Is there something wrong from my end? by the way I have Fullscreen enabled and vsync

**Comment by MGRich** [comment]
Can you post a screenshot?

**Comment by instinctualjealousy** [comment]
This is probably related to @MGRich bringing up bad datafiles. This happened to me with Sonic 1 using the Amazon apk's Data.rsdk, and I got the title screen freeze (on sparkle) with Sonic 2's Data.rsdk. I purchased from Amazon purely because I liked the cleaner approach of buying it outright instead of paying to turn off ads. It might be worth noting in the readme that Amazon's older builds of Sonic 1 and 2 have incompatible datafiles, so while it's a legal source of ownership, you're going to have to source the newer files elsewhere. Of course, others need to confirm this first but it seems to be the case.
